python/sidecar/projects.py

```diff
@@ -67,11 +66,17 @@ def get_project_path(user_id: str, project_id: str) -> Path:
     return Path(data[project_id]["project_path"])
 
 
+def get_project_name(user_id: str, project_id: str) -> Path:
```

Comment:
Why is the name a `Path`? I'd expect a `str`.

Reply:
Bc I just copy/pasted python code! This was fixed already.